### PR TITLE
SvelteKit: README

### PR DESCRIPTION
**Pull request by JReinhold**
Add [documentation](https://github.com/storybookjs/storybook/blob/jeppe/sveltekit-docs/code/frameworks/sveltekit/README.md) for SvelteKit framework.

### CODE REVIEW

File: code/frameworks/sveltekit/README.md [hunk 1]
```diff
@@ -1 +1,106 @@
-# Storybook for SvelteKit
+# Storybook for SvelteKit <!-- omit in toc -->
+
+Our goal is to help you use the tools you love together with Storybook. That’s why Storybook has zero-config support for SvelteKit with the `@storybook/sveltekit` package.
+
+Check out our [Frameworks API](https://storybook.js.org/blog/framework-api/) announcement for what this all means for you and our continuos efforts to make Storybook a seamless integration into any project.
```
Comment (Integrayshaun) [inline code/frameworks/sveltekit/README.md:5]:
```suggestion
Check out our [Frameworks API](https://storybook.js.org/blog/framework-api/) announcement for what this all means for you and our continued efforts to make Storybook a seamless integration for any project.
```

Comment (shilman) [inline code/frameworks/sveltekit/README.md:5]:
```suggestion
Check out our [Frameworks API](https://storybook.js.org/blog/framework-api/) announcement for what this all means for you and our continuous efforts to make Storybook a seamless integration into any project.
```

File: code/frameworks/sveltekit/README.md [hunk 2]
```diff
@@ -1 +1,106 @@
...
+```js
+// .storybook/main.cjs
+module.exports = {
+  ...
+  // shorthand
+  framework: '@storybook/sveltekit',
+  // or if you need options
+  framework: {
+    name: '@storybook/sveltekit',
+    options: {...},
```
Comment (shilman) [inline code/frameworks/sveltekit/README.md:88]: What are the options?

**Review by shilman — APPROVED** [review]
This is awesome @JReinhold !! Left a couple comments

File: code/frameworks/sveltekit/README.md (hunk 1 above)
Comment (JReinhold) [reply code/frameworks/sveltekit/README.md:5]: https://github.com/storybookjs/storybook/pull/20229#discussion_r1047249495

File: code/frameworks/sveltekit/README.md (hunk 2 above)
Comment (JReinhold) [reply code/frameworks/sveltekit/README.md:88]: There are none...

File: code/frameworks/sveltekit/README.md (hunk 1 above)
Comment (JReinhold) [reply code/frameworks/sveltekit/README.md]: Went with Shauns suggestion instead

File: code/frameworks/nextjs/README.md [hunk 3]
```diff
@@ -12,14 +12,18 @@
   - [Next.js's Image Component](#nextjss-image-component)
```
Comment (kylegach) [inline code/frameworks/nextjs/README.md:12]: Thanks for fixing our goofs here, @JReinhold! 🙏

Comment (JReinhold) [reply code/frameworks/nextjs/README.md:12]: Hehe well the VSCode extension did that automatically 😅
https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one

**Pull request merged**